tutorials/src/bin/basic-tutorial-2.rs

```diff
@@ -0,0 +1,59 @@
+extern crate gstreamer as gst;
+
+use gst::GstObjectExt;
+use gst::ElementExt;
+use gst::MessageView;
+
+fn main() {
+    // Initialize GStreamer
+    gst::init().unwrap();
+
+    if let (Some(source), Some(sink)) = (
+    // Create the elements
+    gst::ElementFactory::make("videotestsrc", "source"),
+    gst::ElementFactory::make("autovideosink", "sink")) {
+        // Create the empty pipeline
+        let pipeline = gst::Pipeline::new("test-pipeline");
+
+        // Build the pipeline
+        gst::BinExtManual::add_many(
+            &pipeline,
+            &[&source, &sink]).unwrap();
+        if let Err(_) = gst::ElementExt::link(&source, &sink) {
```

Comment:
`source.link(&sink)` or `gst::Element::link_many(&source, &sink)`

Reply:
Also on error here, you could as well `expect("Failed to link elements)` instead of manually unwrapping the `Result` and exiting the process (same in the other places: element creation for example).